libraries/AP_RCProtocol/AP_RCProtocol_CRSF.cpp

```diff
@@ -276,6 +276,11 @@ bool AP_RCProtocol_CRSF::check_frame(uint32_t timestamp_us)
         return false;
     }
 
+    if (_frame.length + CRSF_HEADER_LEN < _frame_ofs) {
```

Comment:
the first part of your if() is redundent as we check frame_ofs >= CRSF_HEADER_TYPE_LEN above
the 2nd part would avoid this specific bug, but I don't think it is as strong a check as the one in the PR